C-Plus-Plus/cp/remove_spaces_from_a_string.cpp

```diff
@@ -0,0 +1,41 @@
+// Given a string , remove spaces from it and then return new string

```

Comment:
Add both the problem description and your approach in a single multi-line comment.